src/main/java/atmosfears/AtmosFearsBE/database/SensorCode.java

```diff
@@ -0,0 +1,33 @@
+package atmosfears.AtmosFearsBE.database;
+
+import lombok.Getter;
+
+public enum SensorCode {
+    MpKrakAlKras("al. Krasińskiego", 50.057678, 19.926189),
```

Comment:
I would suggest moving this data to a JSON file (see [this](https://github.com/AtmosFears/AtmosFears-BE/pull/15/files#r1217660525) comment).